browser/ui/webui/basic_ui.h

```diff
@@ -13,6 +13,7 @@
 #include "base/compiler_specific.h"
 #include "base/macros.h"
 #include "content/public/browser/web_ui_controller.h"
+#include "brave/browser/version_info.h"
```

Comment:
why is there a header added here and no other code change? Seems like this is in the wrong place and should be in a `cc` file somewhere?

Reply:
removed